packages/modules/event-bus-local/src/services/event-bus-local.ts

```diff
@@ -54,7 +54,7 @@ export default class LocalEventBusService extends AbstractEventBusModuleService
         eventData.name
       )
 
-      if (!eventData.options?.internal) {
```

Comment:
what's the case here?

Reply:
Options can be passed to the emit or as part of the message, so we check both